src/Microsoft.Health.Fhir.Shared.Core.UnitTests/Features/Search/Expressions/Parsers/ExpressionParserTests.cs

```diff
@@ -302,6 +302,7 @@ private SearchParameterInfo SetupSearchParameter(ResourceType resourceType, stri
             {
                 Name = paramName,
                 Type = SearchParamType.String,
+                Url = $"http://testparameter/{resourceType}-{paramName}",
```

Comment:
>Url = $"http://testparameter/{resourceType}-{paramName}", [](start = 16, length = 57)

Do we need to update `SearchParameterInfo` constructor to ensure Url is present?

Reply:
Looking through the code, currently there are quite a few instances where SearchParamterInfo is instantiated with an empty URL.  Mostly test code.  For custom search parameters, we definitely need it populated, but we check for that case.  I am thinking to leave as is for now.